lib/fog/google/requests/storage_json/put_bucket_acl.rb

```diff
@@ -0,0 +1,56 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Mock
+        def put_bucket_acl(_bucket_name, _acl)
+          Fog::Mock.not_implemented
+        end
+      end
+
+      class Real
+        # Change access control list for an Google Storage bucket
+        def put_bucket_acl(bucket_name, acl)
```

Comment:
Unused method argument - `bucket_name`. If it's necessary, use `_` or `_bucket_name` as an argument name to indicate that it won't be used. You can also write as `put_bucket_acl(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `acl`. If it's necessary, use `_` or `_acl` as an argument name to indicate that it won't be used. You can also write as `put_bucket_acl(*)` if you want the method to accept any arguments but don't care about them.